src/styles/blocks/card.scss

```diff
@@ -0,0 +1,74 @@
+.card {
+  width: $CardWidth;
+  padding: ($CardWidth / 100 * 16) ($CardWidth / 100 * 8) ($CardWidth / 100 * 8);
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: $CardWidth * 4 / 5;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: $FontMedium;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    color: $ColorSecondary;
```

Comment:
The variable `$ColorSecondary` should be defined in your variables file or imported correctly. Ensure that it is declared and accessible here.